app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -78,8 +78,16 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
       cirrusCheckStatuses.clear();
       bool runsGoldenFileTests = false;
       // Query current checks for this pr.
-      final List<dynamic> cirrusChecks =
+      final List<CirrusResult> cirrusResults =
           await queryCirrusGraphQL(pr.head.sha, cirrusClient, log, 'flutter');
+      if (!cirrusResults.any(
```

Comment:
So this will only apply the `flutter-gold` status to pre-submit checks that will be landing on master?

When Gold is able to support branching in the future, I think we will want the `flutter-gold` check to work on these branched pre-submit checks. Right now individual pre-submit tests with golden files will not fail since they rely on this `flutter-gold` check. 

Can you add a TODO here to explain that?

Reply:
Yes. It will check only `master` branch based on the above logic.
I see your comment: https://github.com/flutter/flutter/issues/51807#issuecomment-599790674
Do you mind create corresponding issue, and then I will add a TODO pointing to that issue?

Reply:
https://github.com/flutter/flutter/issues/52700 :) Thanks!

Reply:
TODO added.